src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -284,6 +285,95 @@ private void testEditQuestionAction() throws Exception {
         feedbackEditPage.clickAndCancel(feedbackEditPage.getCancelQuestionLink(-1));
     }
     
+    private void testEditVisibilityAction() throws Exception {
+
+        ______TS("edit question 1 to Giver's team members and Giver");
```

Comment:
I'm not sure what you mean. Why is "not displayed" different from "hidden"? They look the same to me, WebElement's <code>isDisplayed</code> seems to work correctly even for that non-standard html element. Can you show me what css selector are you using, and how you are testing whether or not that row is displayed?